tests/integration/docker-compose.yml

```diff
@@ -3,14 +3,33 @@ version: '3.1'
 
 services:
   mysql:
+    container_name: "mysql_master"
+    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    environment:
+      MYSQL_DATABASE: database
+      MYSQL_USER: dbuser
+      MYSQL_PASSWORD: DBpwd1234
+      MYSQL_ROOT_PASSWORD: DBpwd1234
+    build:
+      context: ../../
+      dockerfile: tests/integration/MasterDockerfile
+    links:
+      - nri-mysql
+
+  mysql_slave:
```

Comment:
Is intended for the slave to not have a dependency with the master? Not sure if mysql itself handles this.

Reply:
I added the dependency although the the integration-tests code also controls the availability of both instances